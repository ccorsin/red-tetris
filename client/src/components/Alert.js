import React from 'react';
import styled from 'styled-components';

const StyledAlert = styled.div`
  width: ${props => props.size}vh;
  height: ${props => props.size}vh;
  border-radius: 0px;
  background: rgba(${props => props.color}, 1);

  @media (min-width: 1025px) {
    width: 3vh;
    height: 3vh;
  }
`;

const Alert = ({ message }) => {
  return (
    <StyledAlert>
      <span>{message}</span>
      {/* <button> OK </button> */}
    </StyledAlert>
	);
};

export default Alert;
