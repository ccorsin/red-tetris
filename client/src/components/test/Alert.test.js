import React from 'react'
import { shallow } from 'enzyme';
import Alert from '../Alert';
import toJson from "enzyme-to-json"

describe('<Alert/> Component', () => {
  it('rendering correctly Alert', () => {
    let wrapper = shallow(
      <Alert message='alert' turnOffAlert=''/>
    )
    expect(toJson(wrapper)).toMatchSnapshot()
  })
})