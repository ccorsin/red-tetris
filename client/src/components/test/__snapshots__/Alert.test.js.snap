// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Alert/> Component rendering correctly Alert 1`] = `
<styled.div>
  <styled.button
    onClick={[Function]}
  >
     - 
  </styled.button>
  <styled.div>
    <span>
      alert
    </span>
  </styled.div>
</styled.div>
`;
